modules/doc/content/framework_development/git.md

```diff
@@ -35,6 +37,27 @@ export PS1='$(__git_ps1 "(%s)")$ '
 
 Assuming your prompt is just '$ ', adding this line to your `.bash_profile` will cause it to look like `(master)$` whenever you `cd` into a directory containing a git repo in where the `master` branch is checked out.
 
+#### Using the zsh shell
+
+If your machine uses the zsh shell, two files are required for bash completition. Within the `~/.zsh` directory, download both the git bash completition script and the zsh completition wrapper
+
+```zsh
+curl -o git-completion.bash https://raw.githubusercontent.com/git/git/master/contrib/completion/git-completion.bash
```

Comment:
@cticenhour would you like this curl command in the bash section as well? For folks comfortable with a terminal it is faster, but I wasn't sure what the discussion around the original instructions had been

Reply:
If you want to make things symmetrical like that, I would be for it. But I won't require it (since I think folks who want to use these, like power users, will grab these files from github however they prefer anyway).